exercises/exercise_1/README.md

```diff
@@ -1,336 +1,193 @@
 
-**NOTE:** that the containers are prefixed with the `current directory` then `service` then `replicas index`. The application name is the current directory.
 
-**NOTE:** Hit `Alt+Enter` to enter a *fullscreen* mode
+## The Application
 
-### Deploy using `docker stack`
+For this workshop, we will use a modified version of the [Voting App](https://github.com/dockersamples/example-voting-app). This application contains the following services:
 
-Another way to deploy your compose file is to use `docker stack deploy`. It introduces the concept of stack, which you can query directly from docker.
-During this workshop we will focus on `swarm` orchestrator, but the same commands work the same way targeting `kubernetes` orchestrator.
+- A Python webapp that provides the ability to vote between two options
+- A Redis cache to store new votes
+- A .NET Core worker to consume votes and store them in…
+- A Postgres DB backed by a Docker volume for storing results
+- A Node.js webapp to show the results of the voting in real time
 
-The `docker stack` commands
-```sh
-$ docker stack --help
+![Voting App Architecture](architecture.png)
 
-Usage:  docker stack [OPTIONS] COMMAND
 
-Manage Docker stacks
+## Finishing the Compose File
 
-Options:
-      --orchestrator string   Orchestrator to use (swarm|kubernetes|all)
+The compose file found in the `exercise_1` directory has all of the services needed for the application, but has a few missing pieces to help us learn a little bit more about Docker Compose. We will fill in a few pieces now, but will continue to evolve our compose file as we advance through the workshop.
 
-Commands:
-  deploy      Deploy a new stack or update an existing stack
-  ls          List stacks
-  ps          List the tasks in the stack
-  rm          Remove one or more stacks
-  services    List the services in the stack
 
-Run 'docker stack COMMAND --help' for more information on a command.
-```
+### Helpful Tips for Play-with-Docker
 
-We will now use the following `docker-compose.yml` file, which describes a simple application with 3 services, a frontend, a backend and a database.
-```yaml
-version: "3.7"
-services:
-    web:
-     image: dockerdemos/lab-web
-     ports:
-      - "33000:80"
-
-    words:
-     image: dockerdemos/lab-words
-     deploy:
-       replicas: 3
-       endpoint_mode: dnsrr
-
-    db:
-     image: dockerdemos/lab-db
-```
+- Working within the terminal can be tricky, especially for folks not familiar with CLI-based text editors. To help, use the "Editor" feature in PWD.
+- You can drag-and-drop files from your local machine on to the terminal to easily upload them.
```

Comment:
You can eventually re-use the screenshot `![editor button](editor-button.png)` here?

Reply:
Ah yes! I meant to do that, but forgot to actually do it. I'll add it in.